_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
@nscyclone well. I didn't remember I already added sections.home to every locale. So just use that one. Translators will have to translate a few things after this migration. I guess it is OK to rely on it, otherwise we will have to do , `if lang != 'en' and sections.home == 'Home', then use "project.title"` but that's would be a dirty hack and we don't know if some language borrow the English term Home  . So I would say let's rely on the translation of locale.